local/include/core/http.h

```diff
@@ -683,6 +698,22 @@ class Session
 
   virtual swoc::Rv<std::shared_ptr<HttpHeader>> read_and_parse_request(swoc::FixedBufferWriter &w);
 
+  /**
+   * Peeks at the socket for PROXY header data. Consume from the session socket
```

Comment:
I think the `/**` should be on the same line? Alan told me to keep it on the same line. I'm not sure how important that is.

Reply:
fixed.